jbpm-test-coverage/src/test/java/org/jbpm/test/functional/timer/GlobalTimerServiceVolumeTest.java

```diff
@@ -159,8 +162,9 @@ public void afterNodeLeft(ProcessNodeLeftEvent event) {
     }
 
     @After
-    public void tearDown() {
+    public void tearDown() throws IOException, SQLException {
         globalScheduler.shutdown();
+        createTimerSchema();
```

Comment:
Shouldn't it be `dropTimerSchema` in tearDown?
```suggestion
        dropTimerSchema();
```

Reply:
yes, you're completely right, thanks for that!